orthopairs/src/test/java/org/reactome/release/orthopairs/OrthopairFileGeneratorTest.java

```diff
@@ -30,10 +30,10 @@
     @Mock
     JSONObject mockJSONObject;
 
-    @Before
-    public void setUp() {
-        MockitoAnnotations.initMocks(this);
-    }
+//    @Before
+//    public void setUp() {
+//        MockitoAnnotations.initMocks(this);
+//    }
```

Comment:
Why is the mock init commented out?

Reply:
Its not used in the current tests. It was causing an error when I would run all tests, but when I ran the test individual it was fine

Reply:
Oh, I see - it's just using the collections that aren't mocked.  Got it, thanks!